### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by JorikSchellekens**
image

This does not currently validate the identifiers. That will be added later when the parser lands.

# figma: 
https://www.figma.com/file/WSXjCGc1k6FVI093qhlzOP/04-Recieving-share-link?node-id=59%3A1

matrix to

# demo:

https://matrix-to-2i193unp8.vercel.app/

### CODE REVIEW

File: src/components/Button.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+limitations under the License.
+*/
+
+@import "../color-scheme";
+
+.button {
+  width: 100%;
+
+  padding: 1rem;
+  border-radius: 1000px;
```
Comment (t3chguy) [inline src/components/Button.scss:23]: why 1000?

File: src/components/Button.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  border: 0;
+
+  background-color: $foreground;
+  color: $background;
+
+  font-size: 14px;
+  font-weight: 500px;
```
Comment (t3chguy) [inline src/components/Button.scss:30]: this is invalid according to https://www.w3schools.com/cssref/pr_font_weight.asp

File: src/components/Button.scss (hunk 1 above)
Comment (JorikSchellekens) [reply src/components/Button.scss]: Wasn't sure how this should respond to larger contents but 100px is silly. Using 2rem instead.

**Comment by JorikSchellekens** [comment]
Sorry for the million files. They're all very small

File: package.json [hunk 3]
```diff
@@ -1,57 +1,65 @@
...
+        "@types/react-dom": "^16.9.0",
+        "@types/yup": "^0.29.3",
+        "classnames": "^2.2.6",
+        "formik": "^2.1.4",
+        "react": "^16.13.1",
+        "react-dom": "^16.13.1",
+        "react-scripts": "3.4.1",
+        "typescript": "~3.7.2",
+        "yup": "^0.29.1"
+    },
```
Comment (t3chguy) [inline package.json:22]: A bunch of these feel like they should be devDeps like in react-sdk and riot 

`@testing-*`
`@types/*`
`typescript`

File: src/_error.scss [hunk 4]
```diff
@@ -0,0 +1,4 @@
+@mixin error {
```
Comment (t3chguy) [inline src/_error.scss:1]: copyright

File: src/_color-scheme.scss [hunk 5]
```diff
@@ -4,3 +4,5 @@ $foreground: #000000;
 $font: #333333;
```
Comment (t3chguy) [inline src/_color-scheme.scss:4]: copyright

File: src/components/TextButton.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+*/
+
+import React from "react";
+import classnames from "classnames";
+
+import "./TextButton.scss";
+
+const TextButton: React.FC<React.ButtonHTMLAttributes<Element>> = ({
+    className,
+    ...restProps
```
Comment (t3chguy) [inline src/components/TextButton.tsx:24]: in Input.tsx you call this `...props`, stay consistent with that please

**Comment by JorikSchellekens** [comment]
This resolves: https://github.com/matrix-org/matrix.to/issues/90

**Pull request merged**